src/powerman/arglist.h

```diff
@@ -17,11 +17,16 @@
 #define PM_ARGLIST_H
```

Comment:
In the commit message:

>  Problem: In many cases, there is no way for a power operation (i.e.
power on, but not power status) to inform powerman that an error has
occurred in the operation.  The user will always get a
"Command completed successfully" output and exit status 0 after
issuing a power operation.

That's not really correct.  Powerman was designed with the idea that you could "expect" a string that indicates success.  In the absence of that, there is a timeout and an error.  What this does is add a "fast fail" so you can match an error string and avoid the timeout.

Reply:
I don't think this commit message was fixed?

Reply:
I did tweak it, but lets try again

```
    Problem: Powerman device files can be scripted to "expect" a string to indicate
    success.  In the event of a failure (i.e. an unexpected string output), the device
    script will ultimately time out.  This is undesirable as it can slow down powerman
    when simple errors (e.g. "password failure") fail immediately.  The lack of the
    ability to "fail fast" has lead to many device scripts being scripted to simply "succeed"
    most of the time.  Powerman will complete with "Command completed successfully"
    regardless if the power operation was a success or not.
```